project/config/afbini/config/facets.facet.news_publication_date.yml

```diff
@@ -35,7 +39,7 @@ widget:
     reset_text: 'Show all'
     hide_reset_when_no_selection: false
     soft_limit_settings:
-      show_less_label: 'Show fewer'
+      show_less_label: 'Show less'
```

Comment:
Should this not be fewer or is it changing in Corp lite?

Reply:
It could be changing , Nisra has it set to less